### PR TITLE
Update for Revised P1

### PR DESCRIPTION
**Pull request by jamesjuett**
We're releasing a revised P1 this term. This PR makes a few updates to the tutorial to match.

**P1 Starter Files**
The tutorials show output from `tree` and include several screenshots of various IDEs that happen to show some files. I updated the `tree` output since that was easy and it's nice to have that match. Instead of updating many images (which aren't necessarily focused on the exact starter files), I added a note that "your specific starter files might not match the image" where appropriate.

**Command Line Args instead of Input Redirection**
P1 now uses command line arguments instead of input redirection. So, I moved the command line arguments section before the input redirection section. I also switched the messages at the top of each, so that students are told to skip input redirection for P1.

### CODE REVIEW

File: docs/setup_vscode_macos.md [hunk 1]
```diff
@@ -303,72 +303,72 @@ Open Settings on VSCode (Code > Settings > Settings). Search for "lldb: show dis
...
 <img src="images/vscode_macos_037.png" width="768px" />
 
-### Input redirection
+### Arguments and options
 <div class="primer-spec-callout info" markdown="1">
 Skip this subsection your first time through the tutorial.  You can come back to it.
 </div>
 
-If you're unfamiliar with input redirection, first read the CLI tutorial section on [input redirection](cli.html#input-redirection-).
+Arguments and options are inputs to a program typed at the command line.  Here's an example from EECS 280 Project 5:
```
Comment (amirkamil) [inline docs/setup_vscode_macos.md:311]: Shoulds this be Project 4 now?

File: docs/setup_vscode_wsl.md [hunk 2]
```diff
@@ -373,72 +373,72 @@ Edit the `"environment"` property in your `launch.json`.  If there's already an
...
 {: data-highlight="3-4" }
 
-### Input redirection
+### Arguments and options
 <div class="primer-spec-callout info" markdown="1">
 Skip this subsection your first time through the tutorial.  You can come back to it.
 </div>
 
-If you're unfamiliar with input redirection, first read the CLI tutorial section on [input redirection](cli.html#input-redirection-).
+Arguments and options are inputs to a program typed at the command line.  Here's an example from EECS 280 Project 5:
```
Comment (amirkamil) [inline docs/setup_vscode_wsl.md:381]: Ditto

File: docs/setup_xcode.md [hunk 3]
```diff
@@ -229,11 +229,34 @@ Select your scheme, then "Edit Scheme".  You can also use menu: Product > Scheme
 
 <img src="images/xcode170.png" width="768px" />
 
-### Input redirection
+### Arguments and options
 <div class="primer-spec-callout info" markdown="1">
 Skip this subsection your first time through the tutorial.  You can come back to it.
 </div>
 
+Arguments and options are inputs to a program typed at the command line.  Here's an example from EECS 280 Project 5:
```
Comment (amirkamil) [inline docs/setup_xcode.md:237]: Here too

File: docs/setup_gdb.md [hunk 4]
```diff
@@ -92,17 +92,6 @@ We recommend enabling the address sanitizer and undefined behavior sanitizer. Th
...
-Run with input redirection.  Make sure to add the name of your input file (`main_test.in` in this example).
-```
-$ gdb main.exe
-...
-(gdb) r < main_test.in
-...
-```
-
 ### Arguments and options
 Arguments and options are inputs to a program typed at the command line.  Here's an example from EECS 280 Project 5:
```
Comment (amirkamil) [inline docs/setup_gdb.md:96]: And another

File: docs/setup_lldb.md [hunk 5]
```diff
@@ -91,18 +91,6 @@ We recommend enabling the address sanitizer and undefined behavior sanitizer. Th
...
-```
-$ lldb main.exe
-...
-(lldb) settings set target.input-path main_test.in
-(lldb) r
-...
-```
-
 ### Arguments and options
 Arguments and options are inputs to a program typed at the command line.  Here's an example from EECS 280 Project 5:
```
Comment (amirkamil) [inline docs/setup_lldb.md:95]: One more

File: docs/setup_visualstudio.md [hunk 6]
```diff
@@ -209,54 +209,53 @@ Visual Studio provides an address sanitizer with bounds checking automatically w
...
-### Input redirection
+### Arguments and options
 <div class="primer-spec-callout info" markdown="1">
 Skip this subsection your first time through the tutorial.  You can come back to it.
 </div>
 
-If you're unfamiliar with input redirection, first read the CLI tutorial section on [input redirection](cli.html#input-redirection-).
-
-Without input redirection, here's how to type input in the Visual Studio command line.  Notice that when we run, a window pops up that accepts user input.
+Arguments and options are inputs to a program typed at the command line.  Here's an example from EECS 280 Project 5:
```
Comment (amirkamil) [inline docs/setup_visualstudio.md:217]: Probably should do a `grep` or `sed` for this

File: docs/setup_vscode_macos.md (hunk 1 above)
Comment (jamesjuett) [reply docs/setup_vscode_macos.md]: Fixed via c970e4d.

File: docs/setup_vscode_wsl.md (hunk 2 above)
Comment (jamesjuett) [reply docs/setup_vscode_wsl.md]: Fixed via c970e4d.

File: docs/setup_xcode.md (hunk 3 above)
Comment (jamesjuett) [reply docs/setup_xcode.md]: Fixed via c970e4d.

File: docs/setup_gdb.md (hunk 4 above)
Comment (jamesjuett) [reply docs/setup_gdb.md]: Fixed via c970e4d.

File: docs/setup_lldb.md (hunk 5 above)
Comment (jamesjuett) [reply docs/setup_lldb.md]: Fixed via c970e4d.

File: docs/setup_visualstudio.md (hunk 6 above)
Comment (jamesjuett) [reply docs/setup_visualstudio.md]: Fixed via c970e4d.

**Review by awdeorio — COMMENTED** [review]
LGTM

**Pull request merged**